cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/UninstallUtils.java

```diff
@@ -0,0 +1,120 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.packerina;
+
+import org.ballerinalang.compiler.BLangCompilerException;
+import org.wso2.ballerinalang.compiler.util.ProjectDirConstants;
+import org.wso2.ballerinalang.util.RepoUtils;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.file.Files;
+import java.nio.file.LinkOption;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * This class provides util methods for uninstalling Ballerina packages.
+ *
+ * @since 0.982.0
+ */
+public class UninstallUtils {
+    private static final Path BALLERINA_HOME_PATH = RepoUtils.createAndGetHomeReposPath();
+    private static PrintStream outStream = System.out;
+
+    /**
+     * Uninstalls the package from the home repository.
+     *
+     * @param fullPkgPath package provided
+     */
+    public static void uninstallPackage(String fullPkgPath) {
+        String orgName;
+        String packageName;
+        String version;
+        // Get org-name
+        int orgNameIndex = fullPkgPath.indexOf("/");
+        if (orgNameIndex == -1) {
+            throw new BLangCompilerException("no org-name is provided");
```

Comment:
Is BLangCompilerException the right type of exception here ?

Reply:
Changed the exception type with 1d90337